README.md

```diff
@@ -1,27 +1,43 @@
-This is the Sitecore Developer Portal project.
-It is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+![Sitecore logo in red background](https://raw.githubusercontent.com/Sitecore/developer-portal/Readme/public/sitecore.svg)
```

Comment:
This logo doesn't work for "dark mode" GitHub users as the "Sitecore" text isn't readable.

Maybe we can just use the 'rings' alongside the H1 "Sitecore Developer Portal" heading?